packages/circuit-ui/components/NotificationToast/NotificationToast.spec.tsx

```diff
@@ -30,7 +32,9 @@ import {
 } from './NotificationToast';
 
 describe('NotificationToast', () => {
-  beforeEach(() => jest.clearAllMocks());
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
```

Comment:
In Vitest, hooks can return teardown functions, so implicit returns should be avoided.